uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -917,13 +919,50 @@ private void subscribe(@NonNull String clientDeviceId, @NonNull String topicFilt
         }
     }
 
+    /**
```

Comment:
Idea of test.

MQTT should deliver exactly the SAME message as was published by another client.
To implement that use below approach.

In step/method we got beginning of message text as a string and length of whole message. 
On TX side we call method which produce from that short text and length full message. It should not be random. For example we can join text as many times as need to produce full message, or add filler by template.  After that publish_long_message() will just call another publish() method.


When we going to receive the message we also note in the step the  same string and whole message length. Receive step will call the same method to produce predictable full message and just call another receive() method.

Reply:
fixed in 29f79187174dd5710c02c749df6632e330bc6cf2